src/static/helpers/slasClient.ts

```diff
@@ -109,3 +109,19 @@ export interface ISlasClient {
 
   clientConfig: ClientConfig;
 }
+
+/**
+ * Custom query parameter type with any string prefixed with `c_` as the key and the allowed
+ * types for query parameters for the value.
+ */
+export type CustomQueryParameters = {
+  [key in `c_${string}`]: string | number | boolean | string[] | number[];
+};
+
+/**
+ * Custom body request type with any string prefixed with `c_` as the key and the allowed
+ * types for the value.
+ */
+export type CustomRequestBody = {
```

Comment:
`CustomQueryParameters` and `CustomRequestBody` seems to be the exact same type?

I feel like we had previously added support for `c_` parameters, is there a chance that the type definition is already implemented somewhere in the codebase?

Reply:
We used type `any` when we originally implemented custom parameters, which should be updated to use the `CustomQueryParameters` in the next major version release, I've documented that work in a ticket as part of the OAS spike

`CustomQueryParameters` and `CustomRequestBody` should not be the same type as you pass in complex objects for the body

In `src/static/helpers/fetchHelper.ts` we use `BodyInit`, it also might make sense here